src/appsco-app.html

```diff
@@ -2680,15 +2734,40 @@
                 this._notify('An error occurred. Selected accounts were not removed from company. Please try again.');
```

Comment:
It would be better for code readability to pass false as a second argument (persistant).